http-clients/aws-crt-client/src/main/java/software/amazon/awssdk/http/crt/AwsCrtAsyncHttpClient.java

```diff
@@ -135,9 +146,16 @@ private HttpClientConnectionManager createConnectionPool(URI uri) {
         Validate.notNull(uri, "URI must not be null");
         log.debug(() -> "Creating ConnectionPool for: URI:" + uri + ", MaxConns: " + maxConnectionsPerEndpoint);
 
-        return new HttpClientConnectionManager(bootstrap, socketOptions, tlsContext, uri,
-                                                HttpClientConnectionManager.DEFAULT_MAX_BUFFER_SIZE, windowSize,
-                                                maxConnectionsPerEndpoint);
+        HttpClientConnectionManagerOptions options = new HttpClientConnectionManagerOptions()
+                .withClientBootstrap(bootstrap)
+                .withSocketOptions(socketOptions)
+                .withTlsContext(tlsContext)
+                .withUri(uri)
+                .withWindowSize(windowSize)
+                .withMaxConnections(maxConnectionsPerEndpoint)
+                .withManualWindowManagement(manualWindowManagement);
+
+        return HttpClientConnectionManager.create(options);
```

Comment:
create is a wierd verb for a builder pattern

Reply:
Wouldn't that be a change we would have to make on the CRT side?